db/migrations/20190709160237_create_flip_storage_tables.sql

```diff
@@ -233,11 +260,14 @@ CREATE TABLE maker.flip_kicks
 
 CREATE INDEX flip_kicks_header_id_index
     ON maker.flip_kicks (header_id);
+CREATE INDEX flip_kicks_kicks_index
```

Comment:
I think we need to add a `DROP INDEX flip_kicks_kicks_index;` for this.

Reply:
Huh, idk where that came from - think it popped up in a rebase and I assumed it came from `staging`. Will probably just remove it since we're not exposing this table on this branch 👍